### PR TITLE
UG update for version 1.2

### PR DESCRIPTION
**Pull request by prerthan99**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -75,27 +75,44 @@ Output:
 `3) list`<br>
 `4) help`
 
-### Adding a client's contact: `add`
+### Adding a patinet's contact: `add`
```
Comment (BigDoot) [inline docs/UserGuide.md:78]: Typo on this line

**Review by BigDoot — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 2]
```diff
@@ -110,26 +127,26 @@ Output:
...
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st patient to be `91234567` and `johndoe@example.com` respectively.
+*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd patient to be `Betsy Crower` and clears all existing tags.
 
-### Locating persons by name[Coming soon] : `find`
+### Locating patient by name[Coming soon] : `find`
 
-Finds persons whose names contain any of the given keywords.
+Finds patiets whose names contain any of the given keywords.
```
Comment (vivegank) [inline docs/UserGuide.md:149]: Another one

**Review by vivegank — APPROVED** [review]
LGTM

**Pull request merged**